src/pages/Navbar.js

```diff
@@ -1,15 +1,20 @@
+/* eslint-disable import/no-extraneous-dependencies */
 import React from 'react';
+import { Icon } from '@iconify/react';
 
 const Navbar = () => (
-  <div className="navbar">
-    <div className="nav-title">
-      <h1>BookStore CMS</h1>
+  <header className="navbar">
+    <div>
+      <h1 className="nav-title">BookStore CMS</h1>
     </div>
     <div className="nav-links">
-      <a className="link" href="/">Books</a>
-      <a className="link" href="/Category">Category</a>
+      <a className="books" href="/">BOOKS</a>
+      <a className="categories" href="/Category">CATEGORIES</a>
     </div>
-  </div>
+    <div className="oval">
+      <Icon icon="bi:person-fill" className="mask" />
+    </div>
```

Comment:
- [ ] Please make center the icon inside the circle.
![image](https://user-images.githubusercontent.com/38635158/167259038-4e562270-7bc5-47c2-ba76-648710205b0b.png)